### PR TITLE
(feat) dropped expressJS and implemented pure node

### PR DESCRIPTION
**Pull request by lee-philip-n**
https://trello.com/c/gcPvzpN6
https://trello.com/c/0abWn2y2

### CODE REVIEW

File: db/database.js [hunk 1]
```diff
@@ -1,82 +1,48 @@
...
+    phone: String,
+  },
+  banner: [],
+  photo: [],
+}).index({ name: 1 });
+
+const About = mongoose.model('About', aboutSchema);
+
+const find = (obj, cb) => {
+  About.find(obj).lean().exec((err, about) => {
```
Comment (matgbx) [inline db/database.js]: Does lean make much of a difference in terms of query time?

File: server/node.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    res.writeHead(200, { 'Content-Type': 'text/css' });
+    cssStream.pipe(res);
+  } else if (req.url.match('.js')) {
+    const jsStream = fs.createReadStream(path.join(__dirname, '../client/dist/bundle.js'), 'utf8');
+    res.writeHead(200, { 'Content-Type': 'text/javascript' });
+    jsStream.pipe(res);
+  } else if (req.url.match(/\/restaurants\/Restaurant%20\d{8}$/)) {
+    const id = req.url.slice(26);
+    db.find({ name: `Restaurant ${id}` }, (err, data) => {
+      if (err) {
```
Comment (matgbx) [inline server/node.js]: Might be better to change db.find() to something more specific so it's clear what you are 'finding' and that you are not simply calling Model.find() right here in this file - was a little confusing to follow at first.

**Review by matgbx — CHANGES_REQUESTED** [review]
Nice refactor to node. That all looks good, but suggested on refactor around renaming your database find function for improved clarity and readability

File: db/database.js (hunk 1 above)
Comment (lee-philip-n) [reply db/database.js]: http://www.tothenew.com/blog/high-performance-find-query-using-lean-in-mongoose-2/

File: db/database.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ const aboutSchema = mongoose.Schema({
 
 const About = mongoose.model('About', aboutSchema);
 
-const find = (obj, cb) => {
+const findRestaurant = (obj, cb) => {
```
Comment (matgbx) [inline db/database.js]: looks good!

**Review by matgbx — APPROVED** [review]
All looks good to go now!

**Pull request merged**